scripts/repos.js

```diff
@@ -7,16 +7,17 @@
     var truncatedRepos = repoInfo.all.map(function(ele) {
       return {
         name: ele.name,
-        url: ele.html_url
+        url: ele.html_url,
+        fork: ele.fork
       };
     });
     return truncatedRepos;
   };
 
   repoInfo.getRepos = function(callback) {
     return $.ajax({
-      url: 'https://api.github.com/users/jisenber/repos?per_page=10&sort=updated',
-      tyle: 'GET',
+      url: 'https://api.github.com/user/repos',
+      type: 'GET',
       headers: {'Authorization': 'token ' + githubToken},
       success: function(data) {
```

Comment:
You don't need a header here! also I don't think you are running the middleware correctly. should be `url: 'github/user/repos'` this will be important because it works together with your `app.get` & `proxyGithub` functions in your server.js